wrangler.toml

```diff
@@ -27,6 +27,9 @@ r2_buckets = [
 [env.production.build]
 command = "npm run build"
 
+[env.production.vars]
+MAX_CAR_CIDS_TO_RESOLVE = "250"
```

Comment:
It's tricky because a DAG might be split across 999 CARs but because of the path you might be able to extract `/smol.json` with the last sub-request.

Reply:
With our 10MB CAR shards it'll restrict freeway from serving anything that's over 2.5GB

Reply:
I understand the disadvantage for these use cases. However, at this point looks like the priority should be to reduce R2 ops and avoid potential error states mentioned in issue (break in mid stream without feedback for user, break after worker run out of sub-requests in general). When we have bigger sub-requests limit and better R2 perf we can increase these.

We can tweak more the number if you feel like there is a better compromise. Thoughts?